### ISSUE TITLE
Add composer.json

### ISSUE THREAD

**Issue by tpetry** [GeorgeArgyros/Secure-random-bytes-in-PHP#5]
It would be really nice if you would add an composer.json

**Comment by franzliedke** [comment]
That's true. I had planned to suggest that, too, but then didn't, because I forgot that single files can be autoloaded, too - without any namespacing or PSR-whatever stuff.
